packages/gatsby-cli/src/structured-errors/error-map.ts

```diff
@@ -64,6 +64,17 @@ const errors = {
     type: Type.WEBPACK,
     level: Level.ERROR,
   },
+  "98011": {
```

Comment:
Should we add docsUrl gatsby.dev/typescript

Reply:
There is one below in the actual error message. I think I agree with the current placement of it.